app/components/Image/Image.tsx

```diff
@@ -39,7 +47,11 @@ const ImageComponent = (props: Props) => {
     }
 
     const image = new Image();
-    image.src = src;
+    if (darkModeSource) {
+      image.src = darkModeSource;
+    } else {
+      image.src = src;
+    }
```

Comment:
Wait how is this `darkModeSource` relevant here in this PR? It's a nice feature, but I can't see the relevance. How does a user even upload a dark mode source?

Reply:
This is for static images from /public I believe and just snuck in. There's another issue on dynamic uploadable dark mode images.